src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,5 @@
+package duke;
+
+public class DukeException extends Exception {
```

Comment:
Maybe you could have other exceptions extending from DukeException, such as DukeInvalidTaskException, DukeInvalidCommandException. Although in this case, it is not very important, it might become important later on when the code is evolving.

Reply:
Hi Vignesh, thanks for the insight! Did not realize how messy exceptions would become. Will definitely separate them out :)